trade/trader_binance.js

```diff
@@ -151,37 +155,31 @@ module.exports = (apiKey, secretKey, pwd, log, publicOnly = false, loadMarket =
      * @param {Boolean} nonzero Return only non-zero balances. By default, Binance return zero assets as well.
      * @returns {Promise<unknown>}
      */
-    getBalances(nonzero = true) {
+    async getBalances(nonzero = true) {
       const paramString = `nonzero: ${nonzero}`;
 
-      return new Promise((resolve) => {
-        binanceApiClient.getBalances().then((data) => {
-          try {
-            let result = [];
-
-            data.balances.forEach((crypto) => {
-              result.push({
-                code: crypto.asset,
-                free: +crypto.free,
-                freezed: +crypto.locked,
-                total: +crypto.free + +crypto.locked,
-              });
-            });
-
-            if (nonzero) {
-              result = result.filter((crypto) => crypto.free || crypto.freezed);
-            }
+      try {
+        const data = await binanceApiClient.getBalances();
 
-            resolve(result);
-          } catch (e) {
-            log.warn(`Error while processing getBalances(${paramString}) request: ${e}`);
-            resolve(undefined);
+        try {
+          let result = data.balances.map((crypto) => ({
+            code: crypto.asset,
+            free: +crypto.free,
+            freezed: +crypto.locked,
+            total: +crypto.free + +crypto.locked,
+          }));
+
+          if (nonzero) {
+            result = result.filter((crypto) => crypto.free || crypto.freezed);
           }
-        }).catch((err) => {
-          log.warn(`API request getBalances(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${err}`);
-          resolve(undefined);
-        });
-      });
+
+          return result;
+        } catch (error) {
+          log.warn(`Error while processing getBalances(${paramString}) request: ${error}`);
+        }
+      } catch (error) {
```

Comment:
If you change `catch (err)` ⟶ `catch (error)`, change it everywhere in the module.
Same for `catch (e)`.